docs/tutorials/using_bdk_with_hardware_wallets.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Using BDK with hardware wallets"
+description: "Tutorial showing how to send funds to a HW and then spend from it using BDK"
+authors:
+    - danielabrozzoni
+date: "2022-10-27"
+tags: ["BDK", "Development", "Hardware Wallets"]
+hidden: true
+draft: false
+---
+
+## Introduction
+
+The bitcoindevkit organization maintains [rust-hwi](https://github.com/bitcoindevkit/rust-hwi), a Rust wrapper around Bitcoin Core's [HWI](https://github.com/bitcoin-core/HWI). rust-hwi makes it possible to use hardware wallets with BDK, which is exactly what we're going to do in this tutorial.
+
+## Prerequisites
+
+To follow along you'll need the `hwi` python package installed on your sistem, and a hardware wallet.
+
+Never use a hardware wallet with real funds for testing! Either buy a separate one to be used only for tests, or use a hardware wallet emulator, such as:
+- [Speculos](https://github.com/LedgerHQ/speculos)
+- [Trezor emulator](https://docs.trezor.io/trezor-firmware/core/emulator/index.html)
+- [Coldcard emulator](https://github.com/Coldcard/firmware)
+
+To check if `hwi` is installed, open a python terminal and try to import it:
+```
+$ python3
+Python 3.9.13 (main, May 17 2022, 14:19:07) 
+[GCC 11.3.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import hwilib
+```
+
+If nothing happens, you're set! Instead, if you get a `ModuleNotFoundError`, follow the instructions in [HWI's README.md](https://github.com/bitcoin-core/HWI#bitcoin-hardware-wallet-interface) for installing.
+
+Warning: if you're using MacOS and `virtualenv`, you *may* encounter some problems with `rust-hwi`, as we internally use `PyO3`: https://github.com/PyO3/pyo3/issues/1741
```

Comment:
Typo: funny enough the operating system is spelled _macOS_, without the capital M.